### DISCUSSION TITLE
Partitioned Cookies and Partitioning [Key] : Value

### DISCUSSION THREAD

**Discussion by iepoch** [cypress-io/cypress#29510] [Questions and Help]
Hello, currently today in cypress we can getAllCookies. But when pulling those even in documentation we can pull only so many values.
https://docs.cypress.io/api/commands/getallcookies
And it only yields:
domain: (String)
expiry: (Number) (if specified)
hostOnly: (Boolean) (if specified)
httpOnly: (Boolean)
name: (String)
path: (String)
sameSite: (String) (if specified)
secure: (Boolean)
value: (String)

However if you want to pull PartitionKey it will not pull the partition key no matter if you pass the key of partitionKey. I am wondering when Cypress will update this as we will need to  move to use PartitionKey and values for third party phaseout. This is used in the CHIPS method going forward?

**Comment by portyu9** [comment]
Hi @iepoch 

As of current Cypress, `cy.getAllCookies()` still does **not** expose the CHIPS `partitionKey`.

The returned cookie object currently contains fields such as `domain`, `expiry`, `hostOnly`, `httpOnly`, `name`, `path`, `sameSite`, `secure`, and `value`, but not the cookie's partition key.

So there is currently no supported Cypress cookie API for directly asserting that CHIPS metadata.

If your goal is to test CHIPS behavior, I would test the observable behavior across the relevant top-level sites rather than relying on `partitionKey` metadata. If you specifically need to inspect that metadata, that still requires browser-level tooling outside Cypress's documented cookie API.

Please mark this answer as accepted if it helped, thank you.
